clients/lsp-toml.el

```diff
@@ -0,0 +1,251 @@
+;;; lsp-toml.el --- lsp-mode TOML integration  -*- lexical-binding: t; -*-
```

Comment:
Question: Should I rename it to `lsp-taplo`, `lsp-toml-taplo`, etc. instead of `lsp-toml`?

Reply:
we follow the standard `lsp-<language-name>` so `lsp-toml` looks good to me

Reply:
I see. Thank you.